sysbrokers/IB/config/ib_config_futures.csv

```diff
@@ -320,12 +326,12 @@ MSCICHINA,M1CNX,EUREX,USD,50,1,FALSE
 MSCICHINANET,MCN.HK,HKFE,USD,50,1,FALSE
 MSCIEAFA,MXEA,NYSELIFFE,USD,50,1,FALSE
 MSCIEAFA-TOT,M1EA,CME,USD,20,1,FALSE
-MSCIEM,M1EF,EUREX,USD,50,1,FALSE
+MSCIEM,M1EF,EUREX,USD,100,1,FALSE
 MSCIEM-LIFFE,MXEF,NYSELIFFE,USD,50,1,FALSE
 MSCIEMASIA,M1MSA,NYSELIFFE,USD,100,1,FALSE
 MSCIEURONET,M7EU,EUREX,EUR,100,1,FALSE
 MSCIEURONET-ICE,M7EU0FNN,ICEEU,EUR,100,1,FALSE
-MSCIEUROPE,MXEU,EUREX,USD,10,1,FALSE
+MSCIEUROPE,MXEU,EUREX,EUR,100,1,FALSE
```

Comment:
Here, we know the multiplier (100), but not the pointsize hence not the magnifier. I cannot seem to find it anywhere. Any suggestions?